aconfig/aconfig.py

```diff
@@ -192,7 +192,7 @@ def from_yaml(cls, config_location=None, **kwargs):
         return cls(loaded_config, **kwargs)
 
     @staticmethod
-    def _verify_config_location(config_location):
+    def _verify_config_location(config_location: str):
```

Comment:
```suggestion
    def _verify_config_location(config_location: str) -> bool:
```

Reply:
Note that `_verify_config_location` doesn't return a `bool`, but a `str`.